amazon-s3/7.0/modules/ROOT/pages/amazon-s3-connector-upgrade-migrate.adoc

```diff
@@ -14,6 +14,7 @@ Upgrade Anypoint Connector for Amazon S3 (Amazon S3 Connector) to version 7.x.
 == Changes in This Release
 
 * The AWS SDK is now upgraded to version 2.28.28.
+* The *Copy Object* operation now contains the new *Server Side Encryption*, *Sse Customer Key*, *Sse Kms Key Id*, *Sse Customer Key MD5*, and *Sse Customer Algorithm* fields.
```

Comment:
since this is a change included in 7.0.1 and not 7.0, you can remove this line

Reply:
Sure. Will remove this.